[FIX] l10n_ar_currency_update: tasa de recargo en sincronización de moneda.

pablohmontenegro opened the pull request:
Ticket: 102181
Si ya existe tasa entonces no volvemos a actualizarla.

Explicación de los cambios en los siguientes videos: Primer video (duración 15:42mins): https://drive.google.com/file/d/1qbN1i7LrEKr8pPUR1BTlYzavaJXe0dDg/view

Segundo video (duración 02:23mins):
https://drive.google.com/file/d/111IGWYv2tuJPvFScguhpUI_t5Rm55n_k/view Cambios principales:
1) Previene la sobrescritura de tasas existentes para todas las compañías argentinas (no solo las que tienen recargo) 2) Aplica recargo/porcentaje solo a compañías que lo tienen configurado 3) Mejora la selección de certificado usando self.env.company en lugar de self para evitar problemas con recordsets múltiples

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/782.png)](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/782)

pablohmontenegro committed:
Ticket: 102181
Si ya existe tasa entonces no volvemos a actualizarla.

Explicación de los cambios en los siguientes videos: Primer video (duración 15:42mins): https://drive.google.com/file/d/1qbN1i7LrEKr8pPUR1BTlYzavaJXe0dDg/view

Segundo video (duración 02:23mins):
https://drive.google.com/file/d/111IGWYv2tuJPvFScguhpUI_t5Rm55n_k/view Cambios principales:

1)Previene la sobrescritura de tasas existentes para todas las compañías argentinas (no solo las que tienen recargo)
2) Aplica recargo/porcentaje solo a compañías que lo tienen configurado
3) Mejora la selección de certificado usando self.env.company en lugar de self para evitar problemas con recordsets múltiples
